composer.json

```diff
@@ -6,30 +6,37 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "ext-json": "*",
-        "drupal/core": "^8.9 || ^9"
+        "drupal/core": "^9.2",
+        "drupal/ctools": "^3.7",
+        "drupal/file_link": "2.0.4",
+        "drupal/pathauto": "^1.8",
+        "drupal/token": "^1.10"
```

Comment:
I've found that all these dependencies coming from oe_multilingual which is from `require-dev`. Why we cannot put them within require-dev as well?